rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2627,12 +2627,6 @@ def test_upgrade_db_45_to_46(user_data_dir: 'Path', messages_aggregator):
         assert cursor.execute(
             "SELECT COUNT(*) FROM pragma_table_info('evm_events_info') WHERE name='extra_data'",
         ).fetchone()[0] == 1
-        events_with_extra_data = [row[0] for row in cursor.execute(
```

Comment:
Keep this one in actually. And turn the event to a custom one. To see that the logic works.

Reply:
still can't check the `events_with_extra_data` since that looks at all the existing evm events with extra data. But now it converts the event its checking with `existing_evm_event` and `existing_evm_event_extra_data` to a custom event, so it can still check that one.